api-test/internal/e2e/e2e.go

```
@@ -0,0 +1,731 @@
+			"clientNameLangMap": map[string]any{"eng": "bdd e2e"},
+			"relyingPartyId":    "bdd-e2e-rp",
+			"logoUri":           "https://example.org/logo.png",
+			"redirectUris":      []string{spec.RedirectURI},
+			"publicKey":         publicJWK(priv, kid),
+			"userClaims":        spec.UserClaims,
+			"authContextRefs":   spec.Acr,
+			"grantTypes":        []string{"authorization_code"},
+			"clientAuthMethods": []string{"private_key_jwt"},
+		},
+	}
+	body, _ := json.Marshal(reqBody)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)